.github/workflows/osrm-backend.yml

```diff
@@ -331,7 +331,7 @@ jobs:
             build_node_package: true
             continue-on-error: true
             node: 20
-            runs-on: macos-11
+            runs-on: macos-13 # x86_64
```

Comment:
Looking at  https://github.com/Project-OSRM/osrm-backend/pull/6864 it seems we can use macos-14-large here, right @DennisOSRM ?

Reply:
Yes, that should probably work.